meson_options.txt

```diff
@@ -370,6 +370,18 @@ option('efi-includedir', type : 'string', value : '/usr/include/efi',
        description : 'path to the EFI header directory')
 option('tpm-pcrindex', type : 'integer', value : 8,
        description : 'TPM PCR register number to use')
+option('sbat_distro', type : 'string', value : '',
+       description : 'SBAT distribution ID, e.g. fedora, or auto for autodetection')
+option('sbat_distro_generation', type : 'integer', value : 1,
+       description : 'SBAT distribution generation')
+option('sbat_distro_summary', type : 'string', value : '',
+       description : 'SBAT distribution summary, e.g. Fedora')
+option('sbat_distro_pkgname', type : 'string', value : '',
+       description : 'SBAT distribution package name, e.g. systemd')
+option('sbat_distro_version', type : 'string', value : '',
+       description : 'SBAT distribution package version, e.g. 248-7.fc34')
+option('sbat_distro_url', type : 'string', value : '',
+       description : 'SBAT distribution URL, e.g. https://src.fedoraproject.org/rpms/fwupd')
```

Comment:
The default for type:'string' is '', so please drop `value : ''` everywhere.

All our config options have '-', not '_'. It doesn't matter much, but please change it to '-' for consistency.

All the fields use Fedora systemd package as an example, so I think it'd nicer to use `https://src.fedoraproject.org/rpms/systemd` for the url field too.